docs/_customer-api/_example-key-re-generate-StreamResponseDTO.md

```diff
@@ -0,0 +1,87 @@
+```json
+{
+  "streamId": "626fa59fe4b018f4ecdc092c",
+  "customerId": "6267d661e4b0420e3acced5e",
+  "name": "uran",
+  "apiKey": "1651484065733_118447bc58266f92d412ced8ba1bab78b6cca802ab6c5fd93713751f75f80feab9883e1bbb57bf04ed475fc3d04400ae7be26aaf88e146762a02731d66d5e7dc",
+  "encoderIds": [],
+  "currentMonthStreamTime": 0,
+  "previousMonthStreamTime": 0,
+  "qualityPreset": "CUSTOM",
+  "streamKey": "62e297fce4b08b9e2c49580b",
+  "tokenAuthKey": "626fa5a1e4b018f4ecdc092d",
+  "enableTokenAuth": false,
+  "shouldMonitor": false,
+  "streamingState": "STOPPED",
+  "streamPhase": "LIVE",
+  "streamUrlType": "STATIC",
+  "httpRetry": false,
+  "encoderPlatform": "CLOUDPROXY",
+  "startedStreamingTimeStamp": 0,
+  "videoStreamQualities": [
+    "5dc56c8e68ce4700013fdc8e"
+  ],
+  "audioStreamQualities": [
+    "5d01058b68ce470001e36763"
+  ],
+  "unbranded": false,
+  "settings": {
+    "targetLatency": 3.0,
+    "minRecoveryDelay": 10,
+    "maxRecoveryDelay": 12,
+    "timeout": 1000,
+    "segmentSizeInSeconds": 2.0,
+    "rateChangeSteps": 10.0,
+    "rateDeltaFactor": 1.1,
+    "rateMaxChange": 0.5,
+    "seekThreshold": 3.0,
+    "updateInterval": 500.0,
+    "audioDelay": 0,
+    "videoDelay": 0
+  },
+  "inputCrop": {
+    "cropMethod": "CENTER",
+    "offsetX": 0,
+    "offsetY": 0
+  },
+  "akamai": {
+    "streamId": "2041504",
+    "eventName": "626fa59fe4b018f4ecdc092c",
+    "useIAS": false,
+    "ownAccount": false
+  },
+  "cdn": {
+    "provisioningStatus": "PROVISIONED"
+  },
+  "cloud": {
+    "nrOfCloudRuns": 0,
+    "cpu": 16384,
+    "memory": 16384,
+    "instanceType": "c5.4xlarge",
+    "numberOfEncoders": 0,
+    "defaults": {
+      "videoDeviceId": 0,
+      "audioDeviceId": 0
+    }
+  },
+  "streamUrls": [],
+  "remoteConfigUrl": "https://cdn.playtotv.com/video-encoder-dev/remoteconfigs/626fa59fe4b018f4ecdc092c.json",
+  "remoteConfigUrlLiverySpecific": "https://cdn.liveryvideo.com/remoteconfigs/626fa59fe4b018f4ecdc092c",
+  "analytics": {
+    "samplingPercentage": 100
+  },
+  "remoteConfig": {
+    "updateInterval": 10
+  },
+  "ingestUrls": [
+    "rtmp://ingest.dev.liveryvideo.com/app/626fa59fe4b018f4ecdc092c",
```

Comment:
We should not post the ingestUrls on our documentation site.
Can you replace this with a fake value?